packages/httpsnippet-client-api/src/index.js

```diff
@@ -116,17 +116,33 @@ module.exports = function (source, options) {
 
   let metadata = {};
   if (Object.keys(source.queryObj).length) {
-    const queryParams = source.queryObj;
-
-    Object.keys(queryParams).forEach(param => {
+    Object.keys(source.queryObj).forEach(param => {
       if (authSources.query.includes(param)) {
-        authData.push(buildAuthSnippet(queryParams[param]));
+        authData.push(buildAuthSnippet(source.queryObj[param]));
 
-        delete queryParams[param];
```

Comment:
Not sure why I was doing this hokey property deletion stuff instead of just setting directly to `metadata`... 🤷